xbmc/music/infoscanner/MusicInfoScanner.h

```diff
@@ -74,6 +74,13 @@ class CMusicInfoScanner : public IRunnable, public CInfoScanner
   */
   INFO_RET UpdateArtistInfo(CArtist& artist, const ADDON::ScraperPtr& scraper, bool bAllowSelection, CGUIDialogProgress* pDialog = NULL);
 
+  /*! \brief Get the types of art for an artist or album that are to be
+  automatically fetched from local files during scanning
+  \param mediaType [in] artist or album
+  \return vector of art types that are to be fetched during scanning
+  */
+  static std::vector<std::string> GetArtTypesToScan(const MediaType& mediaType);
```

Comment:
Now used by MusicUtils.cpp I wonder if this would be better moved there, and thus CMusicInfoScanner depending on MusicUtils rather than vica versa?

Reply:
Ya, I thought about it and I like that, but went for the smaller change for initial review.